modules/serverless-rn-1-35-0.adoc

```diff
@@ -0,0 +1,72 @@
+
+* Knative Eventing now supports long-running background jobs. This feature separates resource-intensive or time-consuming tasks from the primary event processing flow, boosting application responsiveness and scalability.
+
+* Autoscaling for Knative Kafka subscriptions is now enhanced with Kubernetes Event-Driven Autoscaling (KEDA) as a Technology Preview (TP). Autoscaling with CMA/KEDA optimizes resource allocation for Kafka triggers and KafkaSource objects, boosting performance in event-driven workloads by enabling dynamic scaling of Kafka consumer resources.
+
+* {ServerlessLogicProductName} now integrates with Prometheus and Grafana to provide enhanced monitoring support.
+
+* {ServerlessLogicProductName} workflows deployed using the `dev` or `preview` profiles are now automatically configured to generate monitoring metrics for Prometheus.
+
+* The Jobs Service supporting service can now be scaled to zero by configuring the `spec.services.jobService.podTemplate.replicas` field in the `SonataFlowPlatform` custom resource (CR).
+
+* {ServerlessLogicProductName} workflows deployed with the `preview` and `gitops` profiles are now automatically configured to send grouped events to the Data Index, optimizing event traffic.
+
+* A more comprehensive list of errors in the workflow definition is now provided, rather than only displaying the first detected error.
+
+* {ServerlessLogicProductName} is now certified for use with PostgreSQL version `15.9`.
+
+* Event performance between {ServerlessLogicProductName} workflows and the Data Index is improved through event batching. Set `kogito.events.grouping=true` to group events. For further optimization, enable `kogito.events.grouping.binary=true` to reduce the size of grouped events with an alternate serialization algorithm. To compress these events, set `kogito.events.grouping.compress=true`, which lowers event size at the cost of additional CPU usage.
+
+* Compensation states are now invoked when a workflow is aborted.
+
+* {ServerlessLogicProductName} now supports configuring the Knative Eventing system to produce and consume events for workflows and supporting services.
+
+* The secret configurations for the Broker and KafkaChannel (Apache Kafka) have been unified.
+
+[id="fixed-issues-1-35-0_{context}"]
+== Fixed issues
+
+* Previously, Horizontal Pod Autoscaler (HPA) scaled down the Activator component prematurely, causing long-running requests against a Knative Service to terminate. This issue is now fixed. The `terminationGracePeriodSeconds` value is automatically set according to the `max-revision-timeout-seconds` configuration for Knative revisions.
+
+* Previously, requests to a Knative Service with a slow back end could time out because the default OpenShift Route timeout was too short. You can now configure the route HAProxy timeout by specifying an environment variable in the Operator Subscription for {ServerlessProductName} as follows:
```

Comment:
```suggestion
* Previously, requests to a Knative Service with a slow back end could time out because the default {product-title} route timeout was too short. You can now configure the route HAProxy timeout by specifying an environment variable in the Operator `Subscription` object for {ServerlessProductName} as follows:
```